Simplenote/build.gradle

```diff
@@ -30,9 +30,9 @@ android {
         if (project.hasProperty("versionName")) {
             versionName project.property("versionName")
         } else {
-            versionName "2.12-rc-1"
+            versionName "2.12-rc-2"
         }
-        versionCode 120
+        versionCode 122
```

Comment:
`2.11.` was `121` so now we need to bump this again to make sure the build differs.